Lesson_16.py

```diff
@@ -0,0 +1,52 @@
+class Manager:
+    def __init__(self, name, salary, department):
+        self.name = name
+        self.salary = salary
+        self.department = department
+
+class Developer:
+    def __init__(self, name, salary, programming_language):
+        self.name = name
+        self.salary = salary
+        self.programming_language = programming_language
+
+class TeamLead(Manager, Developer):
+    def __init__(self, name, salary, department, programming_language, team_size):
+        Manager.__init__(self, name, salary, department)  # Ініціалізація Manager
+        Developer.__init__(self, name, salary, programming_language)  # Ініціалізація Developer
+        self.team_size = team_size  # Додатковий атрибут для TeamLead
```

Comment:
Please check this example how task should be solved. You need to use full power of inheritance
https://github.com/antohaUa/AQA_241007_Hillel/blob/main/Lesson_17_import/00_prev_inheritance.py